gossipd/routing.c

```diff
@@ -1086,39 +1086,39 @@ static struct wireaddr *read_addresses(const tal_t *ctx, const u8 *ser)
 
 bool routing_add_node_announcement(struct routing_state *rstate, const u8 *msg TAKES)
 {
-       struct node *node;
-       secp256k1_ecdsa_signature signature;
-       u32 timestamp;
-       struct pubkey node_id;
-       u8 rgb_color[3];
-       u8 alias[32];
-       u8 *features, *addresses;
-       struct wireaddr *wireaddrs;
-       fromwire_node_announcement(tmpctx, msg,
-                                  &signature, &features, &timestamp,
-                                  &node_id, rgb_color, alias,
-                                  &addresses);
-
-       node = get_node(rstate, &node_id);
-
-       /* May happen if we accepted the node_announcement due to a local
+	struct node *node;
+	secp256k1_ecdsa_signature signature;
+	u32 timestamp;
+	struct pubkey node_id;
+	u8 rgb_color[3];
+	u8 alias[32];
+	u8 *features, *addresses;
+	struct wireaddr *wireaddrs;
+	fromwire_node_announcement(tmpctx, msg,
+	                           &signature, &features, &timestamp,
+	                           &node_id, rgb_color, alias,
+	                           &addresses);
+
+	node = get_node(rstate, &node_id);
+
+	/* May happen if we accepted the node_announcement due to a local
 	* channel, for which we didn't have the announcement hust yet. */
-       if (node == NULL)
-	       return false;
-
-       wireaddrs = read_addresses(tmpctx, addresses);
-       tal_free(node->addresses);
-       node->addresses = tal_steal(node, wireaddrs);
-
-       node->last_timestamp = timestamp;
-       memcpy(node->rgb_color, rgb_color, 3);
-       tal_free(node->alias);
-       node->alias = tal_dup_arr(node, u8, alias, 32, 0);
-
-       replace_broadcast(node, rstate->broadcasts,
-                         &node->node_announce_msgidx,
-                         msg);
-       return true;
+	if (node == NULL)
+		return false;
+
+	wireaddrs = read_addresses(tmpctx, addresses);
+	tal_free(node->addresses);
+	node->addresses = tal_steal(node, wireaddrs);
+
+	node->last_timestamp = timestamp;
+	memcpy(node->rgb_color, rgb_color, 3);
+	tal_free(node->alias);
+	node->alias = tal_dup_arr(node, u8, alias, 32, 0);
+
+	replace_broadcast(node, rstate->broadcasts,
+	                  &node->node_announce_msgidx,
```

Comment:
Similar to `fromwire_node_announcement` case above.

Reply:
done